### ISSUE TITLE
Boolean env vars broken in Python > 3.12

### ISSUE THREAD

**Issue by oponomarov-tu** [phsmith/rundeck_exporter#103]
With the boolean env vars defined as following:

```yaml
spec:
  containers:
    - args:
        - |
          /app/rundeck_exporter.py
      command:
        - /bin/sh
        - -c
      env:
        - name: RUNDECK_SKIP_SSL
          value: "true"
        - name: RUNDECK_PROJECTS_EXECUTIONS
          value: "true"
        - name: RUNDECK_PROJECTS_EXECUTIONS_CACHE
          value: "true"
        - name: RUNDECK_CPU_STATS
          value: "true"
        - name: RUNDECK_MEMORY_STATS
          value: "true"
        - name: RUNDECK_EXPORTER_DEBUG
          value: "false"
```

For Python > 3.12, the following exception is thrown:

```text
│ Traceback (most recent call last):
│   File "/app/rundeck_exporter.py", line 52, in <module>
│     class RundeckMetricsCollector(object):
│   File "/app/rundeck_exporter.py", line 72, in RundeckMetricsCollector
│     default=literal_eval(getenv('RUNDECK_EXPORTER_DEBUG', 'False')),
│             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
│   File "/usr/local/lib/python3.12/ast.py", line 112, in literal_eval
│     return _convert(node_or_string)
│            ^^^^^^^^^^^^^^^^^^^^^^^^
│   File "/usr/local/lib/python3.12/ast.py", line 111, in _convert
│     return _convert_signed_num(node)
│            ^^^^^^^^^^^^^^^^^^^^^^^^^
│   File "/usr/local/lib/python3.12/ast.py", line 85, in _convert_signed_num
│     return _convert_num(node)
│            ^^^^^^^^^^^^^^^^^^
│   File "/usr/local/lib/python3.12/ast.py", line 76, in _convert_num
│     _raise_malformed_node(node)
│   File "/usr/local/lib/python3.12/ast.py", line 73, in _raise_malformed_node
│     raise ValueError(msg + f': {node!r}')
│ ValueError: malformed node or string on line 1: <ast.Name object at 0x7fe68e948110>
```

```bash
$ python -V 
Python 3.12.6
```

Related:
- https://github.com/phsmith/rundeck_exporter/pull/102

**Comment by phsmith** [comment]
Hi @oponomarov-tu.
Thanks for reporting. I've just released a new version (v2.8.3) that fixes this issue.

**Comment by oponomarov-tu** [comment]
Thanks @phsmith for prompt response and fix! Works like a charm 🎉
